Tests/Massive/Massive.cs

```diff
@@ -1,14 +1,11 @@
 ﻿using System;

```

Comment:
Massive is a separate (similar) tool that is being used for comparison purposes only; we should not be editing any of this code (unless it is to update it to the current version)